packages/replay/src/index.ts

```diff
@@ -1 +1,3 @@
 export { Replay } from './integration';
+
+export type {RecordingEvent} from './types'
```

Comment:
Wasn't sure the best way to expose this type for the test

Reply:
Yeah I was also struggling with this and opted to duplicate the type in https://github.com/getsentry/sentry-javascript/pull/7052. If we're comfortable exposing this as public API, I think exporting it is fine and I'll also switch to it in my PR. 
The one reason against exporting it would be breaking changes in rrweb 2.x, if any? But considering how far off it seems that we're upgrading, I'd say it's negligible. WDYT?